churchland_pipeline_python/processing.py

```diff
@@ -646,4 +646,54 @@ def make(self, key):
         self.insert(neuron_keys)
 
         # insert templates
-        self.Template.insert(template_keys)
\ No newline at end of file
+        self.Template.insert(template_keys)
+
+
+# =======
+# LEVEL 2
+# =======
+
+@schema
+class ManuscriptDataIndices(dj.Lookup):
+    definition = """
+    # Collection of indices assigned to neurons, motor units, etc. within a manuscript
+    manuscript_id:    smallint unsigned # manuscript ID number
+    ---
+    manuscript_title: varchar(1012)
+    """
+
+    class MotorUnit(dj.Part):
+        definition = """
+        -> master
+        -> MotorUnit
+        manuscript_motor_unit_id: smallint unsigned # unique within the dataset
+        """
+
+    class Neuron(dj.Part):
+        definition = """
+        -> master
+        -> Neuron
+        manuscript_neuron_id: smallint unsigned # unique within the dataset
```

Comment:
```suggestion
        -> master
        -> Neuron
        ---
        manuscript_neuron_id: smallint unsigned # unique within the dataset
        unique index (manuscript_neuron_id)
```

Reply:
did not realize you could do this -- very cool!